extension/src/popup/views/UnlockAccount.tsx

```diff
@@ -108,15 +100,19 @@ export const UnlockAccount = () => {
                 <HeaderEl>Log in</HeaderEl>
               </HeaderContainerEl>
               <FormRow>
-                <CustomFormTextFieldEl
-                  autoComplete="off"
-                  type="password"
-                  name="password"
-                  placeholder="Enter password"
-                  error={authError}
-                />
+                <Field name="password">
+                  {({ field }: FieldProps) => (
+                    <Input
+                      autoComplete="off"
+                      id="password-input"
+                      error={authError}
+                      placeholder="Enter Password"
+                      type="password"
+                      {...field}
+                    />
+                  )}
+                </Field>
               </FormRow>
-              <ApiErrorMessage error={authError} />
               <ButtonRowEl>
```

Comment:
This should be a pretty good pattern to follow when implementing an SDS Input that's connected to Formik

Reply:
We can remove <ApiErrorMessage /> as SDS Input will handle displaying the error 🎉